util/compobj/user.go

```diff
@@ -454,15 +452,15 @@ func (t CompUsers) checkHash(rule CompUser, shadow []byte) ExitCode {
 		line := scanner.Text()
 		splitedLine := strings.SplitN(line, ":", 3)
 		if splitedLine[0] == rule.User {
-			t.VerboseInfof("user password hash = %s target = %s \n", splitedLine[1], rule.Password)
 			if splitedLine[1] == rule.Password {
+				t.VerboseInfof("for the user %s the hash of the password is %s and should be %s\n", rule.User, splitedLine[1], rule.Password)
 				return ExitOk
 			}
-			t.VerboseInfof("user password hash not ok \n")
+			t.VerboseErrorf("for the user %s the hash of the password is %s and should be %s\n", rule.User, splitedLine[1], rule.Password)
```

Comment:
change to t.VerboseInfof("for the user %s the hash of the password doesn't match expected value\n", rule.User)